src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogConnectionForm.php

```diff
@@ -158,6 +221,32 @@ protected function addConnectionFormatDynamicSubFormListener(FormBuilderInterfac
         $builder->addEventListener(FormEvents::PRE_SUBMIT, $formModificationCallback);
     }
 
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return void
+     */
+    protected function addConnectionTypeDynamicSubFormListener(FormBuilderInterface $builder): void
+    {
+        $connectionTypeSubFormTypes = static::OPTION_CONNECTION_TYPE_SUB_FORM_TYPES;
```

Comment:
Are you sure, that it's the only option to pass constant into callback? Looks very strange.

Reply:
It's the only option I see, that supports https://www.php.net/manual/en/language.oop5.late-static-bindings.php

Do you have any suggestion?